src/main/java/me/partlysanestudios/partlysaneskies/features/gui/CustomMainMenu.java

```diff
@@ -598,40 +593,26 @@ public static void noInfoFound() {
         CustomMainMenu.announcements = new ArrayList<>();
     }
 
-    public static void setFunFact(Request request) {
-        if (!request.hasSucceeded()) {
-            return;
-        }
+    public static void setFunFact() {
+        String url = PartlySaneSkies.Companion.getConfig().getApiUrl() + "/v1/pss/funfact";
```

Comment:
Change to PublicDataManager.getUrl() that way we respect the "get data directly from github"